info.json

```
@@ -6,9 +6,9 @@
     "last_autosave_datetime": "2020-03-03T15:57:40.209367",
     "last_autosave_epoch": 1583243066.897881,
     "name": "someonewhocares.org",
-    "link": "http://someonewhocares.org/hosts/zero/hosts",
+    "link": "https://someonewhocares.org/hosts/zero/hosts",
     "issue_link": "mailto:hosts@someonewhocares.org",
-    "raw_link": "http://someonewhocares.org/hosts/zero/hosts",
+    "raw_link": "https://someonewhocares.org/hosts/zero/hosts",
     "link": "https://someonewhocares.org/hosts/zero/",
     "issue_link": "mailto://hosts@someonewhocares.org",
     "description": "Use this file to prevent your computer from connecting to selected internet hosts. This is an easy and effective way to protect you from many types of spyware, reduces bandwidth use, blocks certain pop-up traps, prevents user tracking by way of "web bugs" embedded in spam, provides partial protection to IE from certain web-based exploits and blocks most advertising you would otherwise be subjected to on the internet.",
```

Can you please fix the issue in this line ? Thanks!

```suggestion
    "description": "Use this file to prevent your computer from connecting to selected internet hosts. This is an easy and effective way to protect you from many types of spyware, reduces bandwidth use, blocks certain pop-up traps, prevents user tracking by way of 'web bugs' embedded in spam, provides partial protection to IE from certain web-based exploits and blocks most advertising you would otherwise be subjected to on the internet.",
```